### PR TITLE
Feature/35540 new dataload

### PR DESCRIPTION
**Pull request by Skeen**
This PR:
* Eliminates the old data-importer in `cli.py` / `bootstrap.py` / `integrations.py`.
* Creates two new endpoints for data-import in `controller.py`, these endpoints are protected by HTTP-BASIC Auth under `OS2PHONEBOOK_DATALOADER_USERNAME` / `OS2PHONEBOOK_DATALOADER_PASSWORD` via the package [`Flask-HTTPAuth`](https://flask-httpauth.readthedocs.io/en/latest/).
* Eliminates `OS2PHONEBOOK_STATIC_ROOT` as content is served by frontend itself.

### CODE REVIEW

File: os2phonebook_service/os2phonebook/controller.py [hunk 1]
```diff
@@ -250,6 +511,8 @@ def call_search_method():
 @api.app_errorhandler(NotFoundError)
 @api.app_errorhandler(InvalidSearchType)
 @api.app_errorhandler(InvalidRequestBody)
+@api.app_errorhandler(InvalidCredentials)
+@api.app_errorhandler(InsufficientCredentials)
 def invalid_validation_handler(error) -> Response:
     """Error handler for all common types
```
Comment (cmoesgaard) [inline os2phonebook_service/os2phonebook/controller.py:518]: Update the docstring below with the new errors (or maybe just remove the bit detailing the error types - it seems a bit redundant)

File: os2phonebook_service/os2phonebook/helpers.py [hunk 2]
```diff
@@ -31,7 +30,6 @@ def config_factory():
 
     required_parameters = [
```
Comment (cmoesgaard) [inline os2phonebook_service/os2phonebook/helpers.py:31]: Should we load the API user/pass variables through here?

File: os2phonebook_service/os2phonebook/controller.py (hunk 1 above)
Comment (Skeen) [reply os2phonebook_service/os2phonebook/controller.py:518]: See 18fe897

File: os2phonebook_service/os2phonebook/helpers.py (hunk 2 above)
Comment (Skeen) [reply os2phonebook_service/os2phonebook/helpers.py:36]: See 02a3550

**Pull request closed without merging**